Scripts/Data/Type/Transformation/FloatToDirection.cs

```diff
@@ -0,0 +1,87 @@
+﻿namespace VRTK.Core.Data.Type.Transformation
+{
+    using UnityEngine;
+    using UnityEngine.Events;
+    using System;
+    using VRTK.Core.Data.Attribute;
+
+    /// <summary>
+    /// Transforms a float value to a distinct rounded direction when the float is outside of the specified deadzone.
+    /// </summary>
+    /// <example>
```

Comment:
I honestly don't get how it works just yet so I don't know if these examples are correct (or whether we should maybe list another set of them for another deadzone).